cylc/uiserver/workflows_mgr.py

```diff
@@ -436,6 +436,7 @@ async def run(self):
             except RuntimeError:
                 # RuntimeError may be raised if the event loop is closed
                 break
+            await asyncio.sleep(0)
```

Comment:
Is this correct? Or should it deliberately block the event loop?

Reply:
I don't think so, the `await` in `await self._queue.get()` should yield control in the same way as `await asyncio.sleep(0)`.